falcon/testing/test_case.py

```diff
@@ -38,30 +40,61 @@ class TestCase(unittest.TestCase):
 
     This base class provides some extra plumbing for unittest-style
     test cases, to help simulate WSGI calls without having to spin up
-    an actual web server. Simply inherit from this class in your test
-    case classes instead of :py:class:`unittest.TestCase` or
-    :py:class:`testtools.TestCase`.
+    an actual web server. Various simulation methods are derived
+    from :py:class:`falcon.testing.TestClient`.
+
+    Simply inherit from this class in your test case classes instead of
+    :py:class:`unittest.TestCase` or :py:class:`testtools.TestCase`.
 
     Attributes:
-        api_class (class): An API class or factory method to use when
-            instantiating the ``api`` instance (default:
-            :py:class:`falcon.API`).
-        api (object): An API instance to target when simulating requests
-            (default: ``self.api_class()``). When testing your
-            application, you will need to overwrite this with your own
-            instance of ``falcon.API``, or use `api_class` to specify a
-            factory method for your application.
+        app (object): A WSGI application to target when simulating
+            requests (default: ``falcon.API()``). When testing your
+            application, you will need to set this to your own instance
+            of ``falcon.API``. For example::
+
+                from falcon import testing
+                from myapp import api
+
+
+                class MyTestCase(testing.TestCase):
+                    def setUp(self):
+                        super(MyTestCase, self).setUp()
+                        self.app = api.create()
```

Comment:
Just hypothetical... let me try to add a comment or something to avoid confusion. 😄